yt_dlp/extractor/patreon.py

```diff
@@ -455,6 +455,23 @@ class PatreonCampaignIE(PatreonBaseIE):
             'thumbnail': r're:^https?://.*$',
         },
         'playlist_mincount': 71,
+    }, {
+        'url': 'https://www.patreon.com/api/campaigns/4767637/posts',
+        'info_dict': {
+            'title': 'Not Just Bikes',
+            'channel_follower_count': int,
+            'id': '4767637',
+            'channel_id': '4767637',
+            'channel_url': 'https://www.patreon.com/notjustbikes',
+            'description': 'md5:595c6e7dca76ae615b1d38c298a287a1',
+            'age_limit': 0,
+            'channel': 'Not Just Bikes',
+            'uploader_url': 'https://www.patreon.com/notjustbikes',
+            'uploader': 'Not Just Bikes',
+            'uploader_id': '37306634',
+            'thumbnail': r're:^https?://.*$',
+        },
+        'playlist_mincount': 71,
```

Comment:
since this is the same test as above, this can be `only_matching`

Reply:
Cool, happy to add that, any chance you can enlighten me on the meaning of `only_matching` so I know what I'm adding?

Reply:
Oh, it's in the contributing guide, missed it. I think I get it.

Reply:
> since this is the same test as above, this can be `only_matching`

No, this is incorrect. This URL format was being matched before this patch, but it was being extracted incorrectly, so we need to test the extraction itself, not just whether the URL matches or not

Reply:
please revert. And ideally we would have a unique test for this URL format instead of duplicating the test data from above

Reply:
Oh sorry I see. My bad, should have read the issue first :)

Reply:
> any chance you can enlighten me on the meaning of `only_matching` so I know what I'm adding?

adding `'only_matching': True` to the test dict will cause this test to be excluded from the download tests and only included in the `TestAllURLsMatching` tests

Reply:
Thank you for the explanation! I replaced the new test I had added with a different new test using a new patreon campaign.